addon/components/radio-button.js

```diff
@@ -27,6 +27,7 @@ export default RadioButtonBase.extend({
       );
       this.set('attributeBindings', updatedAttrs);
       this.get('classNameBindings').pushObject('checked');
+      this.get('classNameBindings').pushObject('disabled');
```

Comment:
Turns out you *do* need this line for it to work in an app, and not just in the unit tests. I'm not sure how to test for this now.